src/Template/Pages/Modules/view.twig

```diff
@@ -1,5 +1,12 @@
 {% do _view.assign('title', currentModule.name|humanize) %}
 
+{% if ((object.id is empty) and (objectType in ['images', 'audio', 'video'])) %}
+<div class="upload-form">
+    <h1>{{ __('Upload {0}', currentModule.singular) }}</h1>
```

Comment:
we don't have `currentModule.singular` right now, we could get that info from `/model/object_types`... remove it for now